ProjectSettings/InputManager.asset

```diff
@@ -293,3 +293,35 @@ InputManager:
     type: 0
     axis: 0
     joyNum: 0
```

Comment:
Because I'm using custom inputs so we can change it if it's needed. Just going into Edit->preferences-> Input you can change the keys used to manage the camera instead of changing the script.